pkg/auth/service.go

```diff
@@ -1027,7 +998,7 @@ func (s *DBAuthService) Authorize(ctx context.Context, req *AuthorizationRequest
 	return &AuthorizationResponse{Allowed: true}, nil
 }
 
-func (s *DBAuthService) ClaimTokenIDOnce(ctx context.Context, tokenID string, expiresAt int64) error {
+func (s *KVAuthService) ClaimTokenIDOnce(ctx context.Context, tokenID string, expiresAt int64) error {
 	tokenExpiresAt := time.Unix(expiresAt, 0)
```

Comment:
Looks unrelated to kv

Reply:
It's an interface method, so we need to implement it

Reply:
Extracting common code to a shared function is better.